.github/workflows/test.yml

```diff
@@ -87,13 +87,13 @@ jobs:
       - name: "Lima: vm1: set up iperf3"
         run: |
           limactl shell vm1 sudo DEBIAN_FRONTEND=noninteractive apt-get install -y iperf3
-          limactl shell vm1 systemd-run --user --unit=iperf3 iperf3 -s
+          limactl shell vm1 sudo systemctl start iperf3.service
       - name: "Lima: vm1: get the IP"
         run: |
           limactl shell vm1 ip -4 -json addr show dev lima0 | jq -r .[0].addr_info[0].local | tee /tmp/vm1_iP
       - name: "Lima: vm1: iperf3 (host -> vm1)"
         run: |
-          iperf3 -c "$(cat /tmp/vm1_ip)"
+          iperf3-darwin -c "$(cat /tmp/vm1_ip)"
```

Comment:
How does this solve the macos-15 issue? 👀

Reply:
I don't think this solves the issue - just another cleanup to unify the way we run host to vm tests, used in test/perf.sh.

Reply:
But I can split the changes to check which change is the fix.

Reply:
Removed iperf3-darwin here:
https://github.com/lima-vm/socket_vmnet/pull/82/commits/283be4690decdd4573547f09f8f5109b9e44a65a

macos-15-large run:
https://github.com/lima-vm/socket_vmnet/actions/runs/11980909877/job/33406141644

Reply:
Actually we don't use this is test/perf.sh - this the way I'm testing manually with vfkit, I need to update test/perf.sh.

Reply:
The job failed without iper3-darwin:
https://github.com/lima-vm/socket_vmnet/actions/runs/11980909877/job/33406141644

Maybe it uses different DNS resolver?

Reply:
Thanks, the test doesn't seem relying on DNS?

Reply:
Right, this must be something else.